src/IK/include/BipedalLocomotion/IK/GravityTask.h

```diff
@@ -169,25 +189,6 @@ class GravityTask : public IKLinearTask
      */
     bool update() override;
 
-    /**
-     * @brief Set the desired gravity direction expressed the target frame.
-     * @param desiredGravityDirection The desired gravity direction.
-     * @return True in case of success, false otherwise.
-     *
-     * The input is normalized, unless the norm is too small.
-     */
-    bool setDesiredGravityDirectionInTargetFrame(
-        const Eigen::Ref<const Eigen::Vector3d> desiredGravityDirection);
-
-    /**
-     * @brief Set the feedforward angular velocity expressed in the target frame.
-     * @param feedforwardVelocity The desired feedforward velocity in rad/s.
-     * @return True in case of success, false otherwise.
-     * Only the first two components are used.
-     */
-    bool
-    setFeedForwardVelocityInTargetFrame(const Eigen::Ref<const Eigen::Vector3d> feedforwardVelocity);
-
```

Comment:
This is a breaking change. Is it possible to deprecate first? This would break https://github.com/ami-iit/yarp-device-haptic-gloves/blob/7e30ff3ad8bd4cbed7bedc20963e383200a41080/WeartGlove/src/WeArtGlove.cpp#L750-L753

Reply:
The new check in update will fail and so the ik would not be solved. Do you have any suggestions to how to deal with it?

Reply:
Mitigated in https://github.com/ami-iit/bipedal-locomotion-framework/pull/939/commits/e04d27fd569d147fd8a92d32e147bd2c51d25111

Reply:
Thanks!